domains/api.jamied132.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Subdomain for the api of my website",
+  "owner": {
+    "username": "JamieD132",
+    "email": "jamied132.email@gmail.com",
+    "darflen.com": "JamieD132"
+  },
+  "record": {
+    "NS": ["", ""]
```

Comment:
You know you can try and add the domain to Cloudflare and from there to get the NS records, right?

Reply:
uhhh no i probably don't know that, i am quite new to this :/ what do you mean? Do you mean add jamied132.is-a.dev to cloudflare and from once ive done that give it a api. subdomain?

Reply:
Add the main subdomain to Cloudflare, get the NS records and add the NS records to your **main domain**, not your API subdomain.

Reply:
ok...

Reply:
did i do it right? https://github.com/is-a-dev/register/pull/16899